### PR TITLE
chore: upgrade gradle to version 6

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

https://docs.gradle.org/6.0-rc-3/userguide/upgrading_version_5.html#changes_6.0

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,7 +31,6 @@ allprojects {
     apply plugin: 'java'
     apply plugin: 'jacoco'
     apply plugin: 'eclipse'
-    apply plugin: 'findbugs'
```
Comment (ChristianMurphy) [inline build.gradle]: findbugs plugin was removed in Gradle 6
https://docs.gradle.org/6.0-rc-3/userguide/upgrading_version_5.html#the_findbugs_plugin_has_been_removed

File: build.gradle [hunk 2]
```diff
@@ -219,10 +210,11 @@ release {
 
 task jacocoAggregateReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
     dependsOn = subprojects.test
-    additionalSourceDirs = files(subprojects.sourceSets.main.allSource.srcDirs)
-    sourceDirectories = files(subprojects.sourceSets.main.allSource.srcDirs)
-    classDirectories = files(subprojects.sourceSets.main.output)
-    executionData = files(subprojects.jacocoTestReport.executionData)
+    getAdditionalSourceDirs().setFrom(files(subprojects.sourceSets.main.allSource.srcDirs))
```
Comment (ChristianMurphy) [inline build.gradle]: these properties have been changed to getters and setters
https://docs.gradle.org/6.0-rc-3/userguide/upgrading_version_5.html#other_deprecated_behaviors_and_apis

File: build.gradle [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
     // Top level plugins
-    id 'build-announcements'
     id 'com.github.kt3k.coveralls' version '2.8.4'
-    id 'com.moowork.node' version '1.3.1'
+    id 'com.github.node-gradle.node' version '2.2.0'
     id 'net.researchgate.release' version '2.8.1'
     // Records all tasks in task graph, generates `build/reports/visteg.dot`
     // dot file can be converted to an image using graphviz. `dot -Tsvg -O -v visteg.dot`
-    id 'cz.malohlava.visteg' version '1.0.5'
+    // id 'cz.malohlava.visteg' version '1.0.5'
```
Comment (ChristianMurphy) [inline build.gradle]: this plugin does not yet support Gradle 6

**Comment by jonathanmtran** [comment]
At first blush, things appear to still build and not break.

I've run `./gradlew clean install`. I didn't see anything that broke the build. 
Then I bumped the version in `uPortal-start'`s `gradle.properties`, and `./gradlew :overlays:uPortal:tomcatDeploy`.

**Pull request merged**